cfgov/jinja2/v1/events/event.html

```diff
@@ -1,8 +1,8 @@
 {% extends 'layout-2-1-bleedbar.html' %}
 {% import '_vars-events.html' as vars with context %}
 {% import 'templates/streamfield-sidefoot.html' as streamfield_sidefoot with context %}
-{% if page.start_dt and page.end_dt %}
-    {% set event_state = when(page.start_dt | date('%c'), page.end_dt | date('%c')) %}
+{% if page.live_stream_date and page.end_dt %}
+    {% set event_state = when(page.live_stream_date, page.end_dt) %}
```

Comment:
@jimmynotjim not exactly event_state is also used to determine event description text https://github.com/cfpb/cfgov-refresh/blob/45fb6f96ce4f760bbbe0fb1a10708e9474ce21ec/cfgov/jinja2/v1/events/_event-detail.html#L31

this will effect events that don't have live streams